vllm/worker/cpu_model_runner.py

```diff
@@ -578,6 +615,22 @@ def list_loras(self) -> Set[int]:
         return self.lora_manager.list_adapters()
 
 
+@contextmanager
+def _set_global_compilation_settings():
```

Comment:
context managers need try-catch in them, the yield is not enough to reset the previous state.

Reply:
Nice catch, updated :)